artiq/gateware/targets/kc705_dds.py

```diff
@@ -220,7 +220,12 @@ def __init__(self, cpu_type="or1k", **kwargs):
             self.submodules += phy
             rtio_channels.append(rtio.Channel.from_phy(
                 phy, ofifo_depth=128, ififo_depth=128))
-
+            
+        phy = spi.SPIMaster(mmc_spi)
```

Comment:
@mntng you need to do a ``platform.request()``, ``mmc_spi`` is just a regular Python local variable, it won't appear there just because you added it to the Migen platform file.